walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+:showcase-app-link: https://github.com/aerogear/ionic-showcase
+:integreatly-name: Managed Integration
+:mobile-services-name: Mobile Developer Services
+
+= Building secure, data first mobile app solutions with Mobile Developer Services 
+
+This walkthrough demonstrates how to build a showcase app that uses all the mobile services.
+
+{mobile-services-name} simplifies mobile and modern application development, leveraging OpenShift's container technology to provide a secure, scalable backend platform. 
+The showcase app that you run in this walkthrough demonstrates:
+
+* building an Ionic Framework and Apache Cordova client side application
+
+* creating a Data Sync service. 
++
+The Data Sync framework allows you create apps that perform real-time data synchronization and have offline capabilities.
+In this walkthrough, you use the showcase app code which allows many users to create and edit a set of Tasks items.
+
+* adding authentication and authorization to your mobile app using the Identity Management service
+
+* monitoring the showcase app using the Mobile Metrics services
++
+This service allows you gather metrics on mobile apps, device versions, device security checks and back-end mobile service usage.
+
+.Deployment
+
+The showcase app includes a Node.js server side application that implements an example GraphQL API and connects to a Postgres database.
+
+image::images/artifacts.png[integration, role="integr8ly-img-responsive"]
+
+[type=walkthroughResource, serviceName=openshift]
+.OpenShift Project
+****
+* link:{openshift-host}/console/project/{walkthrough-namespace}/overview[OpenShift Project, window="_blank"]
+****
+
+[type=walkthroughResource]
+.Mobile Developer Console
+****
+* link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
+****
+
+[type=walkthroughResource]
+.Mobile Developer Services Documentation
+****
+* link:https://access.redhat.com/documentation/en-us/red_hat_mobile_developer_services/1[Documentation, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=codeready]
+.CodeReady Workspaces
+****
+* link:https://developers.redhat.com/products/codeready-workspaces/overview/[Overview, window="_blank"]
+* link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces_for_openshift/1.0.0/[Documentation, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=openshift]
+.Red Hat OpenShift
+****
+* link:{openshift-host}/console[Console, window="_blank"]
+* link:https://help.openshift.com/[Openshift Online Help Center, window="_blank"]
+* link:https://blog.openshift.com/[Openshift Blog, window="_blank"]
+****
+
+
+// NOTES: Till all services will be avaiable as operators inside openshift cluster we will require additional installation steps
```

Comment:
Not sure if that should be part of the Walkthrough or we going to have it as prerequisite.